prod.json

```diff
@@ -0,0 +1,4 @@
+{
+  "project": "terra-bard-prod",
+  "samRoot": "https://sam.dsde-prod.broadinstitute.org"
+}
```

Comment:
since this code is a repeat of `config.json`, do we want to do a symbolic link instead?